signerserver/application.py

```diff
@@ -180,10 +180,12 @@ def forward():
             "headers": dict(response.headers),
         }
         try:
+
             proxy_response_dict["response"]["content"] = response.content.decode('utf-8')
-            proxy_response_dict["response"]['json_content'] = response.json()
+            if proxy_response_dict["response"]["content"] != "":
```

Comment:
Why not:
```
if not proxy_response_dict["response"]["content"]:
```
It seems like you're covering more exceptions with the above.

Reply:
Exception is thrown if the content is "". We are preventing the exception being thrown

Reply:
What about when proxy_response_dict["response"]["content"] is None?

Reply:
proxy_response_dict["response"]["content"] is never None. Got it.